docs/src/adr/005adr-json.md

```diff
@@ -70,23 +140,20 @@ OperEx(
 Since both sub-expressions, the literals `1`, are identical, their serializations are equal as well:
 ```
 {
-  kind: "ValEx",
-  type: "Int",
-  value: {
-    kind: "TlaInt",
-    value: 1
+  "kind": "ValEx",
+  "type": "Int",
+  "value": {
+    "kind": "TlaInt",
+    "value": 1
   }
 }
 ```
 
-Note that in the above example, the outer `value` field contains a JSON serialization of `TlaInt(1)`, and not just the JSON literal `1`. This is because of deserialization:
-Apalache encodes its TLA+ integers as `BigInt`, which means that it permits values, for which `.isValidInt` does not hold. If we chose to encode values directly as JSON numbers, we would need a) a sensible encoding of `BigInt` values, which are not valid integers and b) a way to distinguish both variants of `BigInt`, as well as decimals, when deserializing (since JSON is not typed).
-Alternatively, we could encode all values as strings, which would be similarly indistinguishable when deserializing. Importantly, the `type` field is not guaranteed to contain a hint, as it could be `Untyped`.
-For the above reasons, we choose to instead serialize `TlaValue` as a JSON object, which is more verbose, but trivial to deserialize. It has the following shape
+Observe that we choose to serialize `TlaValue` as a JSON object, which is more verbose, but trivial to deserialize. It has the following shape
 ```
 {
-  kind: _
-  value: _
+  "kind": _ ,
+  "value": _
```

Comment:
what are the underscores? Is this standard JSON semantics? Shall we use the syntax for comments like `/* the kind */` and `/* the value /*`.

Reply:
The underscores are, at least to me, very obvious meta-language placeholders. No less formal than `/* */`, since JSON doesn't support comments anyway.

Reply:
Well, I surely understand what you mean by `_`, but we are writing an ADR. So we should not assume too much about the reader. I have not seen the underscore notation outside of OCaml, Scala, and maybe Perl.